content/articles/how-to-create-an-android-wear-os-app/index.md

```diff
@@ -0,0 +1,269 @@
+
+```xml
+<manifest xmlns:android="https://schemas.android.com/apk/res/android"
+    package="com.demo.wear">
+
+    <uses-feature android:name="android.hardware.type.watch" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@android:style/Theme.DeviceDefault">
+
+        <uses-library
+            android:name="com.google.android.wearable"
+            android:required="true" />
+
+        <!-- If your software is Standalone, it means it doesn't need to be run on a mobile device. -->
+
+        <meta-data
+            android:name="com.google.android.wearable.standalone"
+            android:value="true" />
+
+        <activity
+            android:name=".MainActivity"
+            android:label="@string/app_name">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
+```
+### Most common Wear OS UI components
+There are four UI elements in the toolkit that deserve special attention because they are critical for Wear OS development.
+
+- **WearableLinearLayoutManager** allows you to scroll through a list using the watch's mechanical wheels. Furthermore, it moves objects that are positioned along the screen borders to the center, which is highly useful when employing circular interface. 
+
+- **BoxInsetLayout** is an Android FrameLayout capable of adapting child items to a circular display. It positions them in the rectangle region defined by the screen circle. 
+
+Such transformations are ignored at the system level for square displays. As a result, the layout will be similar across all watch interfaces.
+
+**WearableRecyclerView** is a useful design that's widely used in mobile apps, but it's been tailored for the watch in this case. The top and bottom Views may be truncated along the edges due to the display's curved edges, thus this pattern is used to address the problem.
+
+**EdgeItemsCenteringEnabled** makes you capable of creating a curved views for items that are scrolled and improve the components of code, making it easier to view on a relatively small screen.
+
+SwipeDismissFrameLayout is another popular layout. It makes it possible to swipe from left to right. 
+
+### Code example using the `BoxInsetLayout`
+
+```xml
+<androidx.wear.widget.BoxInsetLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    android:padding="8dp"
+    tools:deviceIds="wear"
+    tools:contect=".MainActivity">
```

Comment:
I can spot a typo and I assume you wanted to write `context` instead of `contect`. Please fix that and test your code to ensure that it works.